TOMP-API.yaml

```diff
@@ -1784,65 +1866,83 @@ components:
+          example: Island Central
+        coordinate:
+          $ref: '#/components/schemas/coordinate'
+        physicalAddress:
+          $ref: '#/components/schemas/address'
+        crossStreet:
+          type: string
+          description: Cross street of where the station is located. This field is intended to be a descriptive field for human consumption. In cities, this would be a cross street, but could also be a description of a location in a park, etc.
+          example: on the corner with Secondary Road
+        regionId:
+          type: string
+          description: ID of the region where the station is located (see "systemRegions")
+        serviceArea:
+          $ref: '#/components/schemas/polygon'
+        rentalMethods:
           type: array
+          description: Array of enumerables containing the payment methods accepted at this station.
           items:
-            type: object
-            required:
-              - station-id
-              - name
-              - coordinates
-            properties:
-              station-id:
-                type: string
-                description: unique identifier of a station
-                example: XX:Y:12345678
-              name:
-                type: string
-                description: public name of the station
-                example: Island Central
-              coordinates:
-                $ref: '#/components/schemas/coordinates'
-              physical-address:
-                $ref: '#/components/schemas/address'
-              cross-street:
-                type: string
-                description: Cross street of where the station is located. This field is intended to be a descriptive field for human consumption. In cities, this would be a cross street, but could also be a description of a location in a park, etc.
-                example: on the corner with Secondary Road
-              region-id:
-                type: string
-                description: ID of the region where the station is located (see "system-regions")
-              rental-methods:
-                type: array
-                description: Array of enumerables containing the payment methods accepted at this station.
-                items:
-                  type: string
-                  enum: [KEY, CREDITCARD, PAYPASS, APPLEPAY, ANDROIDPAY, TRANSITCARD, ACCOUNTNUMBER, PHONE, OTHER]
-                example: [CREDITCARD,PAYPASS,APPLEPAY]
-              rental-url:
-                type: string
-                format: URL
-                example: https://www.rentmyfreebike.com
-   
-    stop-reference:
+            type: string
+            enum: [KEY, CREDITCARD, PAYPASS, APPLEPAY, ANDROIDPAY, TRANSITCARD, ACCOUNTNUMBER, PHONE, OTHER]
+          example: [CREDITCARD,PAYPASS,APPLEPAY]
+        rentalUrl:
+          type: string
+          format: URL
+          example: https://www.rentmyfreebike.com
+        numBikesAvailable:
```

Comment:
In GBFS these fields are not in station information but in station status. I am wholly in favour of adding available-assets on a station basis, but I don't think this live data should be in this otherwise quite static file, especially if it's only the reduced functionality from GBFS

Reply:
Do we lack of the /operator/station-status endpoint?

Reply:
Yes, in a sense. It's slightly more complicated than just having missed it.

GBFS has two _files_ for available assets:
`free_bike_status.json` which contains free-floating available bikes
`station_status.json` which has a list of stations and the number of bicycles at each
We have one analogue
`/operator/available-assets` which returns an array of asset types, both free-floating and at a station without any option to sort or limit it to a location. the assets contained in the typeOfAsset object do of course have a place that tells you where it is located

It seems like a good idea to have a station specific `/available-assets` like we do for legs, but the bike-specific station-status doesn't seem like a good implementation of that to me, since the free_bike_status file was also too bike-specific for us. Therefore I'd remove it here and create an issue to add a robust station based available assets path, which allows for all types of assets.